pkg/storage/s3/storage.go

```diff
@@ -143,8 +159,10 @@ func (is *ObjectStorage) initRepo(name string) error {
 
 // InitRepo creates an image repository under this store.
 func (is *ObjectStorage) InitRepo(name string) error {
+	lockStart := time.Now()
+
 	is.Lock()
```

Comment:
Move the time.Now() into is.Lock() {...} itself

So line #162 here becomes "var lockLatency time.Time"
and is.Lock() becomes "is.Lock(&lockLatency)"